legend-engine-executionPlan-execution-store-relational-connection-tests-api/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/tests/api/DynamicTestConnection.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api;
+
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.RelationalDatabaseConnection;
+
+public interface DynamicTestConnection
```

Comment:
Please add some docs that explain what this interface is for.

Implementations have to return a specification of database connections that they might be creating dynamically/on the fly etc.

Reply:
added